uobtheatre/addresses/test/test_model.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from uobtheatre.addresses.test.factories import AddressFactory
+
+
+@pytest.mark.django_db
+def test_address_to_string():
+    address = AddressFactory()
+
+    assert str(address) == f"{address.street}, {address.city}, {address.postcode}"
```

Comment:
function -> f"{address.street}, {address.city}, {address.postcode}"
test -> f"{address.street}, {address.city}, {address.postcode}"
assert function == test

I know this is a super trivial example but still probably worth doing some examples instead

```
@pytest.mark.django_db
def test_str_discount():
    discount = DiscountFactory(discount=0.12, name="student")
    assert str(discount) == "12.0% off for student"
```